src/bindings/jna/libelektra4j/src/main/java/org/libelektra/exception/KDBException.java

```diff
@@ -0,0 +1,90 @@
+package org.libelektra.exception;
+
+import org.libelektra.Key;
+import org.libelektra.exception.model.WarningEntry;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+/**
+ * Custom KDB exception class being used for I/O errors
+ */
+public abstract class KDBException extends java.io.IOException {
+
+	private static final long serialVersionUID = 1L;
```

Comment:
?

Reply:
Same reason as above

Reply:
Where above?

Reply:
The previous author added this for serialization versioning (which was also not very intuitive for me). I will just delete it

Reply:
Yes, I do not know which use case the previous author had in mind for serializing the errors.